lib/text/ttml_text_parser.js

```diff
@@ -386,18 +388,20 @@ shaka.text.TtmlTextParser = class {
     if (origin) {
       percentage = TtmlTextParser.percentValues_.exec(origin);
       results = percentage || TtmlTextParser.pixelValues_.exec(origin);
-      if (results != null) {
-        if (globalHeight != null) {
-          region.viewportAnchorX = Number(results[1]) * 100 / globalHeight;
-        } else {
-          region.viewportAnchorX = Number(results[1]);
-        }
-        if (globalWidth != null) {
-          region.viewportAnchorY = Number(results[2]) * 100 / globalWidth;
-        } else {
-          region.viewportAnchorY = Number(results[2]);
+      if (results !== null) {
+        region.viewportAnchorX = Number(results[1]);
+        region.viewportAnchorY = Number(results[2]);
+
+        if (!percentage) {
+          if (globalHeight !== null) {
+            region.viewportAnchorX = region.viewportAnchorX * 100/globalHeight;
```

Comment:
Style nit: add spaces around the division.

Also, please run the linter and compiler.  I'm making plans for GitHub Actions to check these things for you automatically when you upload a PR, but for now, you can save yourself some review time by running them yourself.

```sh
python build/check.py  # linter, basic type checks, including tests
python build/build.py  # compiler only, no tests
python build/all.py  # everything at once
```

Reply:
whitespace fixed